Scripts~/windows/.clean-download-files.ps1

```diff
@@ -0,0 +1,27 @@
+$TEMP_PATH="C:\Temp"
```

Comment:
I wonder whether we should have another subfolder. Maybe "C:\Temp\GameLiftUnityPlugin".

Reply:
Added a sub folder and updated the cleanup script to delete the whole folder instead of finding specific artifacts